liquibase-standard/src/main/java/liquibase/analytics/configuration/AnalyticsArgs.java

```diff
@@ -8,7 +8,7 @@
 
 public class AnalyticsArgs implements AutoloadedConfigurations {
 
-    private static final ConfigurationDefinition<Boolean> ENABLED;
+    public static final ConfigurationDefinition<Boolean> ENABLED;
```

Comment:
Does this need to be public? I purposefully made it private so that you use the `isAnalyticsEnabled()` method instead, which has additional logic in it.

Reply:
I didn't want to hard code the key for this value, but I suppose I can. It feels better to use `AnalyticsArgs.ENABLED.getKey()` over the string `liquibase.analytics.enabled` IMO, but lmk what you'd prefer.

Reply:
I guess I don't feel strongly about it, I see where you're coming from. We definitely do hardcode strings in other places in the maven plugin (yet another reason it needs to be refactored). I'll leave it up to you about whether you want to hardcode the string in the maven plugin, hardcode the strings in public variables in the AnalyticsArgs class (and then keep the `ENABLED` instance variable private and use those strings in the same class), or keep it public and add a comment so that people know not to use it directly, but to use the other method instead.

Reply:
I am more in favor of Alex's approach. This kind of strings are indeed hardcoded in many places in the Maven plugin, but it would be maybe one less place to refactor later.

Reply:
Cool. I'll add some javadocs to the `ENABLED` arg that says to use the method for checking analytics enabled.